### DISCUSSION TITLE
Welcome to waka-readme-stats Discussions!

### DISCUSSION THREAD

**Discussion by anmol098** [anmol098/waka-readme-stats#320] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eooeo** [comment]
Hi, Anmol! I like your "waka-readme-stats". Thank you for your job.
I have a few questions: 
1. how to remove or replace icons in the sections "I am the night" and "I am most productive on ..."  
2. how to hide the line "...last update"  
3. is it possible to change the width of the "SECTION:waka" - so that the sections "I am the night" and "I am most productive on ..." are next to each other

**Reply by pseusys** [reply]
Hi, @eooeo!
Sorry for the late reply. Considering your questions...
1. You can set `SHOW_COMMIT` and/or `SHOW_DAYS_OF_WEEK` flags to `False`. Please, consult [readme](https://github.com/anmol098/waka-readme-stats#flags-available) file for further info.
2. You can set `SHOW_UPDATED_DATE` flag to `False`.
3. I'm afraid, that's not currently possible.

**Comment by Ash-codes18** [comment]
I have followed the readme, and done what it stated but after running the workflow although the run is successful it doesn't update my readme.
My wakatime profile is only 1 day old, could it be because of that?
Can someone please suggest as to what the problem might be.
Thank you.

**Reply by pseusys** [reply]
As you can see, you've [got a 401 HTTP code](https://github.com/Ash-codes18/Ash-codes18/actions/runs/4105604422/jobs/7082701001#step:3:26) executing user info request. 401 means `Unauthorized`, so I would start doublechecking your GH_TOKEN and GitHub Actions permissions.

**Reply by Ash-codes18** [reply]
> As you can see, you've [got a 401 HTTP code](https://github.com/Ash-codes18/Ash-codes18/actions/runs/4105604422/jobs/7082701001#step:3:26) executing user info request. 401 means `Unauthorized`, so I would start doublechecking your GH_TOKEN and GitHub Actions permissions.

Hi!, Thanks for the suggestion!, As you said there was indeed an problem with the GH_TOKEN and after fixing it, it started working, but now the workflow is getting failed all of a sudden. 

It will be very helpful if you could provide a suggestion as to why this might be happening.

![image](https://user-images.githubusercontent.com/112748790/219655260-0df2e7b8-19d6-4033-b9e7-561ee7c71300.png)

**Reply by overgeared-coder** [reply]
> > As you can see, you've [got a 401 HTTP code](https://github.com/Ash-codes18/Ash-codes18/actions/runs/4105604422/jobs/7082701001#step:3:26) executing user info request. 401 means `Unauthorized`, so I would start doublechecking your GH_TOKEN and GitHub Actions permissions.
> 
> Hi!, Thanks for the suggestion!, As you said there was indeed an problem with the GH_TOKEN and after fixing it, it started working, but now the workflow is getting failed all of a sudden. 
> 
> It will be very helpful if you could provide a suggestion as to why this might be happening.
> 
> ![image](https://user-images.githubusercontent.com/112748790/219655260-0df2e7b8-19d6-4033-b9e7-561ee7c71300.png)
> 


It's working now!, Thanks 🙏🏻

**Comment by StasonMendelso** [comment]
Hi, @anmol098!
Thank you for your work! I want to try this metrics but have the next problem :( : 
![image](https://user-images.githubusercontent.com/124992452/219340579-d00804f0-c130-45d8-8d48-884abf6d37fe.png)
I'm a newbie in github and my account created 5 day ago. Can you help me? 
Have a nice day

**Reply by pseusys** [reply]
Hi there!  
Your bug is surely enough related to [this issue](https://bugs.launchpad.net/ubuntu/+source/python-tz/+bug/1995864) (and this is, in turn, might be related to the city of Kyiv recent renaming, correct me if I'm wrong).  
The best solution in this case in my opinion will be removing many of this repo dependencies and switching to python standard ones (where possible).
But I won't introduce any more changes to this repo until three of my recent pull requests (#352, #363, #365) will be merged - otherwise I'll have to do far too much work resolving merge conflicts.  
Currently I'm waiting for these PRs to be reviewed and approved by this repo owners.
Not much I can help you until them :/

**Reply by StasonMendelso** [reply]
I will wait approving your PR and then try again because I don't know how to fix it. You're right, the correct cname of the capital of Ukraine is ```Kyiv```, not as it was before ```Kiev```
Thank you for answer!

**Reply by pseusys** [reply]
I guess, the easiest quick fix will be temporarily setting location of your profile to some other city in the same time zone.

**Reply by StasonMendelso** [reply]
Unfortunately, after changing the time zone or location of the profile - it did not fix the error. I tried to run another job that [sets the time zone](https://github.com/marketplace/actions/set-timezone), but the job still takes the time zone of ```Kiev``` and throws an exception :(. But now the stacktrace is another 
![image](https://user-images.githubusercontent.com/124992452/219573701-74d10b10-77a7-4cc4-9548-62d9180ec11c.png)
Is it possible to somehow change the Python version if this error occurs because the capital of Ukraine was spelled incorrectly?

**Reply by pseusys** [reply]
#371 changes `pytz` version to `2022.7`, that has the correct name of Kyiv in its database.

**Reply by StasonMendelso** [reply]
Unfortunately, the error is the same. I've written the next `anmol098/waka-readme-stats@feat/code_` and the error was thrown the same :(

**Reply by pseusys** [reply]
The image wasn't published to DockerHub yet - the issue will be resolved as soon as the PR will be merged

**Reply by StasonMendelso** [reply]
Hello, @pseusys! Today I have pushed the last version from master to my github readme repository and now it works. 
Thank you for your efforts!

**Reply by Andriy123421** [reply]
ohhh its hard

**Comment by StasonMendelso** [comment]
Hello everyone!
I have an insteresting problem. The action works, but the result is incredible strange. I have got total around 262 commits (I've found it by using GitHub stats), but the waka-readme-stats drew a very big number (see the figure).
![image](https://user-images.githubusercontent.com/124992452/228457956-d0f888a7-0895-4e9a-9317-12fae16e71a3.png)
I've summed up all commit and then divided by the real number of my commits - I have had a multiplier, which equals 8. Maybe that can help. I don't understand why the code generates these incredible number of commits and need a help. Before that everything was working fine. Another problem is that, the number of Profile views is 0 now, but before was 68.

**Reply by StasonMendelso** [reply]
@pseusys I have the same problem after few months

**Reply by pseusys** [reply]
Could you please include action run with debug logging output?

**Reply by StasonMendelso** [reply]
@pseusys 
I don't know how enable debug level in logging output for action, but can this archive help you? It's logs after my last action's running. 
[logs_4.zip](https://github.com/anmol098/waka-readme-stats/files/12251209/logs_4.zip)
If you can help me enable debug level, I can send another logs.

**Reply by pseusys** [reply]
Nope, that won't help. In your actions section, click `rerun all jobs` and tick `enable debug logging`.

**Reply by StasonMendelso** [reply]
[logs_10.zip](https://github.com/anmol098/waka-readme-stats/files/12291394/logs_10.zip)

**Reply by pseusys** [reply]
@StasonMendelso, could you please check PR #469 out? I think I found a bug in commit data fetching and analysation.
I can not, however, approve neither disprove the enormous commit decrease after the bug fix. If the PR doesn't fix your issue, could you please elaborate on your commit multiplier theory? Where have you found your **true** commit count? Have you checked if the multiplier _exactly_ equals 8? Please, keep in mind, that there can be different ways to calculate commit count: you can include (or not) private repositories, repositories you don't own, but contributed to, etc. Maybe this action just uses other commit counting algorithm?

**Reply by pseusys** [reply]
@StasonMendelso please, refer to [this](https://github.com/anmol098/waka-readme-stats/discussions/466#discussioncomment-6697872) comment that describes the exact algorithm of commit and repository detection and calculation.

**Comment by Egesiapres** [comment]
Hy @anmol098 an thanks for this amazing tool! My answer is: is it possible to hide the timeline through a parameter in the .yml file?

**Comment by ChunhThanhDe** [comment]
Hello @anmol098 ,  Can the SHOW_PROJECTS flag be calculated for Day Projects?

**Reply by ChunhThanhDe** [reply]
![image](https://github.com/user-attachments/assets/adb437c8-906e-4fb4-9373-a7288cecf355)

I want to display the time I worked on the project in a day, how can I setup to do that?

**Comment by Sierraki** [comment]
"Hi, I have already set up my README file and my workflow is running properly, but my README is not being updated. Why is that?"
![Snipaste_2025-05-21_09-29-37](https://github.com/user-attachments/assets/ad1467ad-049e-4f88-9027-d494d3663540)
![Snipaste_2025-05-21_09-29-31](https://github.com/user-attachments/assets/82589a3e-6309-4bd8-ac1f-63eb547b4bb5)

**Reply by ok-coder1** [reply]
@Sierraki Replace those two lines with
```md
<!--START_SECTION:waka-->
<!--END_SECTION:waka-->
```

**Reply by Sierraki** [reply]
Yes, because English is not my native language, I had some problems in understanding. Now the problem has been solved. Thank you.
